### ISSUE TITLE
TS2769: No overload matches this call.

### ISSUE THREAD

**Issue by mary-ng5519** [react-native-elements/react-native-elements#3485]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Explain what you did

Implement the UI with components ```<ListItem> <ListItem.Title>``` on React-native Typed
```
<ListItem.Content>
                    <ListItem.Title style={styles.itemText}>
                      {item.text}
                    </ListItem.Title>
                  </ListItem.Content>
```

### Expected behavior

The component implementation should be compatible with latest TS Script version
```
"@babel/core": "^7.12.9",
    "@babel/runtime": "^7.12.5",
    "@react-native-community/eslint-config": "^2.0.0",
    "@types/jest": "^26.0.23",
    "@types/react": "^18.0.7",
    "@types/react-dom": "^18.0.2",
    "@types/react-native": "^0.67.6",
    "@types/react-test-renderer": "^17.0.1",
    "babel-jest": "^26.6.3",
    "babel-loader": "^8.2.5",
    "babel-plugin-react-native-web": "^0.17.7",
    "eslint": "^7.14.0",
    "html-webpack-plugin": "^5.5.0",
    "jest": "^26.6.3",
    "metro-react-native-babel-preset": "^0.66.2",
    "react-native-typescript-transformer": "^1.2.13",
    "react-scripts": "^5.0.1",
    "react-test-renderer": "17.0.2",
    "source-map-loader": "^3.0.1",
    "ts-loader": "^9.2.8",
    "typescript": "^4.4.4",
    "url-loader": "^4.1.1",
    "webpack": "^5.72.0",
    "webpack-cli": "^4.9.2",
    "webpack-dev-server": "^4.8.1"
```

### Describe the bug

```
     TS2769: No overload matches this call.
  Overload 1 of 2, '(props: ListItemTitleProps, context?: any): ReactElement<any, any> | null', gave the following error.       
    Type '{ children: string; style: any; }' is not assignable to type 'IntrinsicAttributes & ListItemTitleProps'.
      Property 'children' does not exist on type 'IntrinsicAttributes & ListItemTitleProps'.
  Overload 2 of 2, '(props: ListItemTitleProps): ReactElement<any, any> | null', gave the following error.
    Type '{ children: string; style: any; }' is not assignable to type 'IntrinsicAttributes & ListItemTitleProps'.
      Property 'children' does not exist on type 'IntrinsicAttributes & ListItemTitleProps'.
```

### Steps To Reproduce

```markdown
npm install latest package "react-native-elements": "^3.4.2",
```

### Your Environment

```markdown
"react-native-elements": "^3.4.2",
```
```
Binaries:
    Node: 16.13.2 - D:\Program Files\nodejs\node.EXE
    Yarn: 1.22.4 - D:\Roaming\npm\yarn.CMD
    npm: 6.14.5 - D:\Roaming\npm\npm.CMD
    Watchman: Not Found
 npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.2 => 17.0.2
    react-native: ^0.68.1 => 0.68.1
    react-native-windows: Not Found
```
```

**Comment by arpitBhalla** [comment]
Can you try it with latest release?

**Comment by mary-ng5519** [comment]
Sure Dear, latest release still happens that issue

**Comment by mary-ng5519** [comment]
> Can you try it with latest release?

Sure Dear, latest release still happens that issue

**Comment by arpitBhalla** [comment]
Try it with [Bleeding Edge ](https://reactnativeelements.com/docs/next/installation) version

**Comment by MultidimensionalLife** [comment]
Same issue with `<Card />` component. Any updates on this?

**Comment by arpitBhalla** [comment]
@sequencer08 did you tried with `@rneui/themed` or `react-native-elements`?

**Comment by MultidimensionalLife** [comment]
Hi @arpitBhalla I used @rneui/themed. I'm using rne version 4.0.0-rc.5.

For the preview: 

<img width="583" alt="image" src="https://user-images.githubusercontent.com/38305614/177335589-5d63c45a-e473-4543-a3c4-3e821ff3b559.png">

**Comment by abdelhalim97** [comment]
Hi,
i m facing the same issue i have no types or interfaces still having the same almost same bug
"typescript": "3.7.2"
"react": "16.11.0",
updating react made it worst and updating the ts did nothing 

<img width="583" alt="image" src="https://i.ibb.co/rmTpRBb/bbbs-titre.png" style="max-width: 100%;">
it gives error on size props when the props and its value exists in the material-ui docs(v4)
does it have anything to do with vs td version?
<img src="https://i.ibb.co/qRsh4Lm/v.png" alt="v" border="0">

**Comment by arpitBhalla** [comment]
Fixed by #3663
